src/models/methods/pathFind.ts

```diff
@@ -0,0 +1,48 @@
+import { Amount, Path } from '../common'
+import { BaseRequest, BaseResponse } from './baseMethod';
+
+interface BasePathFindRequest extends BaseRequest {
+    command: "path_find"
+    subcommand: string
+}
+
+interface PathFindCreateRequest extends BasePathFindRequest {
+    subcommand: "create"
+    source_account: string
+    destination_account: string
+    destination_amount: Amount
+    send_max?: Amount
+    paths?: Path[]
+}
+
+interface PathFindCloseRequest extends BasePathFindRequest {
+    subcommand: "close"
+}
+
+interface PathFindStatusRequest extends BasePathFindRequest {
+    subcommand: "status"
+}
+
+export type PathFindRequest = PathFindCreateRequest | PathFindCloseRequest | PathFindStatusRequest
+
+interface PathOption {
+    paths_computed: Path[]
+    source_amount: Amount
+}
+
+export interface PathFindResponse extends BaseResponse {
+  result: {
+      alternatives: PathOption[]
+      destination_account: string
+      destination_amount: Amount
+      source_account: string
+      full_reply: boolean
+      id: number | string
```

Comment:
Will we need to update the documentation when 2.0 comes out to limit id's to numbers/strings? (It seems websocket supports any unique object, but we have a convention that is partially applied)

Reply:
`id` should be optional IIRC, and a member of `BaseRequest`/`BaseResponse`

```ts
id?: number | string
```

I think `id` can be anything (so we could type it as `any`), and rippled just echoes it back to you, but I dont really see the harm in restricting it to a `number | string`.

Reply:
@natenichols in this specific case id is echoed in the response, in addition to the id? at the top level in BaseRequest/BaseResponse